packages/tools/src/stateManagement/annotation/annotationState.ts

```diff
@@ -40,58 +39,59 @@ function getViewportSpecificAnnotationManager(
   return defaultFrameOfReferenceSpecificAnnotationManager;
 }
 
+function getAnnotationManager(element?: HTMLDivElement) {
+  if (element) {
+    return getViewportSpecificAnnotationManager(element);
+  }
+
+  return getDefaultAnnotationManager();
+}
+
 /**
- * Returns the annotations for the `FrameOfReference` of the `Viewport`
- * being viewed by the cornerstone3D enabled `element`.
+ * Returns the annotations for a given tool in the specified frame of reference.
+ * If element is provided, it will return the annotations for the viewport specific
+ * annotation manager.
  *
- * @param element - The HTML element.
  * @param toolName - The name of the tool.
+ * @param FrameOfReferenceUID - The Frame of Reference UID.
+ * @param element - The HTML element.
  * @returns The annotations corresponding to the Frame of Reference and the toolName.
  */
 function getAnnotations(
-  element: HTMLDivElement,
-  toolName: string
+  toolName: string,
+  FrameOfReferenceUID: string,
```

Comment:
What FOR is used if not specified by an instance?

Reply:
previously it was getting the FOR from the enabledElement, so nothing is changed here except providing the FOR itself instead of `const {FOR} = getEnabledElement(elemnet)` 

SeriesUID does not seem to be right since a CT and PT can have different UID but still in the same FOR. I don't think you can even use cs3d without FOR